CODEOWNERS

```diff
@@ -1,4 +1,5 @@
 /*.ts                  @SmartBear/bugsnag-dev
 /*.json                @SmartBear/bugsnag-dev
-/common                @SmartBear/bugsnag-dev
+/common                @SmartBear/bugsnag-dev, @pactflow/engineering-team
```

Comment:
`,` is not valid syntax for codeowners file

Reply:
fix pr was raised here https://github.com/SmartBear/smartbear-mcp/pull/119